lib/pf/UnifiedApi/Controller/Nodes.pm

```diff
@@ -0,0 +1,65 @@
+package pf::UnifiedApi::Controller::Nodes;
+
+=head1 NAME
+
+pf::UnifiedApi::Controller::Nodes -
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::UnifiedApi::Controller::Nodes
+
+=cut
+
+use strict;
+use warnings;
+use Mojo::Base 'pf::UnifiedApi::Controller::Crud';
+use pf::dal::node;
+use pf::locationlog qw(locationlog_history_mac locationlog_view_open_mac);
+
+has dal => 'pf::dal::node';
+has id_key => 'id';
+has resource_id => 'pid';
```

Comment:
@jrouzierinverse 

Is the resource id really the pid ?

Reply:
@jrouzierinverse 

Could you confirm if this needs to be changed?

Reply:
It needs to be changed will do so after the merging in feature/api-restructure